_posts/2020-10-05-acidanthera-october.md

```diff
@@ -0,0 +1,153 @@
+---
+layout: post
+title:  "Acidanthera Updates: October 2020"
+date:   2020-10-05 15:00:00 -0700
+categories: Hackintosh updates
+---
+
+* [A message from vit9696](#a-message-from-vit9696)
+* [Guide Updates](#guide-updates)
+* [Changelogs](#changelogs)
+
+# A message from vit9696
+
+We are very pleased to meet you this evening to present our latest achievements on the scene, this update is particularly remarkable for one reason: Today OpenCore becomes the only openly available bootloader capable of running any version of macOS released for Intel CPUs spanning as early as macOS 10.4 to macOS 11, which is currently in late beta phase. Most importantly, the architecture we created allowed us to perform a very clean integration both code-wise and configuration-wise, allowing the user to have a single configuration for every macOS version ever needed. My congratulations to all who made this possible.
+
+Last month we stated that collaboration is a primary goal for Acidanthera in every aspect, and today we present our next steps in this direction. If you use multiple macOS versions you might have noticed that some kexts may not always work on an older operating system if built with a newer Xcode version. This is because the macOS kernel SDK is version dependent and does not always respect the API and ABI differences in the kernel. Previously we used various workarounds available in most Acidanthera kexts which let us target macOS 10.8 and sometimes even lower, and now these workarounds are available to everyone as a standalone product: [MacKernelSDK](https://github.com/acidanthera/bugtracker). This product lets you build kernel extensions with basically any recent Xcode version for basically any macOS version. It is very easy to integrate in your environment and all the kexts in Acidanthera have already been migrated providing a great example to numerous popular third-party kexts migrating right now.
```

Comment:
```suggestion
Last month we stated that collaboration is a primary goal for Acidanthera in every aspect, and today we present our next steps in this direction. If you use multiple macOS versions you might have noticed that some kexts may not always work on an older operating system if built with a newer Xcode version. This is because the macOS kernel SDK is version dependent and does not always respect the API and ABI differences in the kernel. Previously we used various workarounds available in most Acidanthera kexts which let us target macOS 10.8 and sometimes even lower, and now these workarounds are available to everyone as a standalone product: [MacKernelSDK](https://github.com/acidanthera/bugtracker). This product lets you build kernel extensions with basically any recent Xcode version for basically any macOS version. It is very easy to integrate in your environment and all the kexts in Acidanthera have already been migrated, providing a great example to numerous popular third-party kexts migrating right now.
```